README.md

```diff
@@ -197,9 +191,9 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] [OPTIONAL] Please kindly remove all the placeholder text describing all you need to do in each header of the readme file. this should be reasons for invalidating your project. But I made optional because other reviewers did not raised it. Kindly implement this changes.

Reply:
thank you for your feedback and help @brainconnect93 . Sure, I Will do it right now before continuing on the next step